File: app/src/main/java/com/durbar/bangabandhuplay/data/model/live/LiveStreamingResponse.java
package com.durbar.bangabandhuplay.data.model.live;

import androidx.annotation.Nullable;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LiveStreamingResponse {
    @Nullable
    @SerializedName("status")
    @Expose
    public Boolean status;
    @Nullable
    @SerializedName("message")
    @Expose
    public String message;
    @Nullable
    @SerializedName("data")
    @Expose
    public Data data;
    @Nullable
    @SerializedName("errors")
    @Expose
    public Object errors;

    public LiveStreamingResponse(Boolean status, String message, Data data, Object errors) {
        this.status = status;
        this.message = message;
        this.data = data;
        this.errors = errors;
    }

    public Boolean getStatus() {
        return status;
    }

    public String getMessage() {
        return message;
    }

    public Data getData() {
        return data;
    }

    public Object getErrors() {
        return errors;
    }

    @Override
    public String toString() {
        return "LiveStreamingResponse{" +
                "status=" + status +
                ", message='" + message + '\'' +
                ", data=" + data +
                ", errors=" + errors +
                '}';
    }
}

File: app/src/main/java/com/durbar/bangabandhuplay/data/model/live/LiveStreamingResponse.kt
package com.durbar.bangabandhuplay.data.model.live

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class LiveStreamingResponse(
    @field:SerializedName("status")
    val status: Boolean? = null,
    @field:SerializedName("message")
    val message: String? = null,
    @field:SerializedName("data")
    val data: Data? = null,
    @field:SerializedName("errors")
    val errors: Any? = null
)
